packages/api/src/command/webhook/devices.ts

```diff
@@ -57,3 +57,40 @@ export const sendProviderConnected = async (
     });
   }
 };
+
+/**
+ * Sends an update to the CX about their user disconnecting from a provider.
+ * This will also be fired when the user's token is no longer valid and needs to be
+ * programatically .
+ *
+ * Executed asynchronously, so it should treat errors w/o expecting it to be done upstream.
+ */
+export const sendProviderDisconnected = async (
+  connectedUser: ConnectedUser,
+  providers: string[]
+): Promise<void> => {
+  let webhookRequest;
+  try {
+    const { id: userId, cxId } = connectedUser;
+    const providers = connectedUser?.providerMap ? Object.keys(connectedUser.providerMap) : [];
+    const payload = { users: [{ userId, providers, connectedProviders: providers }] };
```

Comment:
We have the same `providers` on diff properties, I think they should be different, no?

Likely b/c there's a param and a local constant with the same name?

Reply:
Good catch